sotodlib/site_pipeline/calibration/wiregrid.py

```diff
@@ -396,6 +395,7 @@ def get_cal_gamma(tod, wrap_aman=False):
     ----------
         tod : AxisManager
         wrap_aman : (default) False
```

Comment:
I'm a bit confused by this. Why do you return a different set of values in the tuple returned compared to what you wrap into the AxisManager when wrap_aman = True?

Reply:
True. That's my bad. Thanks.